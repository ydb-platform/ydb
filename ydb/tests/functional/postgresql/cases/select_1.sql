--
-- BOOLEAN
--

--
-- sanity check - if this fails go insane!
--
SELECT 1 AS one;