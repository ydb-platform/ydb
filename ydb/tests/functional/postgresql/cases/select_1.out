--
-- BOOLEAN
--
--
-- sanity check - if this fails go insane!
--
SELECT 1 AS one;
 one 
-----
   1
(1 row)
